Tasks/NuGetAuthenticateV1/make.json

```diff
@@ -0,0 +1,22 @@
+{
+    "rm": [       
+        {
+            "items": [
+                "node_modules/azure-pipelines-tasks-artifacts-common/node_modules/azure-pipelines-task-lib"
+            ],
+            "options": "-Rf"
+        }
+    ],
+    "externals": {
+        "archivePackages": [
```

Comment:
The url and dest here changed and NuGetAuthenticateV1 is different from NuGetAuthenticateV0. Also, we neew two zip URLs here because the net 6 cred provider is released as a separate asset for now.